packages/builder/src/builderStore/store/workflow/index.js

```diff
@@ -4,8 +4,8 @@ import Workflow from "./Workflow"
 
 const workflowActions = store => ({
   fetch: async instanceId => {
-    const WORKFLOWS_URL = `/api/${instanceId}/workflows`
-    const workflowResponse = await api.get(WORKFLOWS_URL)
+    const WORKFLOWS_URL = `/api/workflows`
+    const workflowResponse = await api(instanceId).get(WORKFLOWS_URL)
```

Comment:
Is the point of the instanceID change not that we are going to stop passing it around? We should just set the instanceID in either localstorage/cookie when we log in or use the builder, then send it automatically in the `api.js` module. Then we don't need to change the code everywhere an API call is made to use this

Reply:
ah yea, this code is old now - i no longer need this. thanks

Reply:
Will this `instanceId` break things here? It looks like the `api` module doesn't take this argument any more.